plugins/groovy

```diff
@@ -0,0 +1 @@
+repository = https://github.com/weibemoura/asdf-groovy
```

Comment:
Can you add `.git` to the end of these URLs? The actual Git repo is `https://github.com/weibemoura/asdf-groovy.git`

Reply:
I sent the changes to a new commit